pkg/kubectl/cmd/get/get.go

```diff
@@ -311,14 +396,21 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 		fmt.Fprintf(o.IOStreams.ErrOut, "warning: --%s requested, --%s will be ignored\n", useOpenAPIPrintColumnFlagLabel, useServerPrintColumns)
 	}
 
+	chunkSize := o.ChunkSize
+	if o.Sort {
+		// if sorting, disable chunked requests - it does not make sense to
+		// sort each chunk individually
```

Comment:
this isn't really true. we could still fetch chunks and then sort them at the end.  chunking and sorting are orthogonal.  I would recommend leaving the comment as a "TODO: in the future we could use chunking to get all results and then sort them at the end, which reduces server load"

Reply:
Added TODO for this. Will open issue to track

Reply:
https://github.com/kubernetes/kubernetes/issues/68027